File: corpus/javascript/fixture_02.js
export function calculateDiscount2(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 20 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook2 {
  lookupPrice(sku) { return sku.length + 2; }
}

File: corpus/python/fixture_02.py
def calculate_discount_2(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 20:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook2:
    def lookup_price(self, sku):
        return len(sku) + 2
